Packs/DemistoRESTAPI/Integrations/CoreRESTAPI/CoreRESTAPI.js

```diff
@@ -177,7 +177,7 @@ var addPlaybookMetadataToRequest = function(body, command) {
 };
 
 
-var sendRequest = function(method, uri, body, raw) {
+var sendRequest = function(method, uri, body, raw, timeout = (3 * 60 * 1000)) {
```

Comment:
It is better to make a const with a descriptive name and comment in code that it is 3 minutes
(in addition to the yml explenation )